src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/PropagateCancellationTokensWhenPossibleTests.cs

```diff
@@ -4,11 +4,266 @@
 using Microsoft.CodeQuality.VisualBasic.Analyzers.ApiDesignGuidelines;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Test.Utilities;
+using Xunit;
 
 namespace Microsoft.CodeQuality.Analyzers.ApiDesignGuidelines.UnitTests
 {
     public class PropagateCancellationTokensWhenPossibleTests : DiagnosticAnalyzerTestBase
     {
+        [Fact]
+        public void WipCSharp_CancellationTokenOverloadAvailable()
```

Comment:
>Wip [](start = 20, length = 3)

All these tests will have Wip in the name.

Reply:
Yeah, I did that so I could just search 'Wip' in Test Explorer in order to run them. I forgot to remove them.

Reply:
Note that test explorer can filter by class name.